### PR TITLE
Add global continental chlorine (pCl and HCl) emissions

### PR DESCRIPTION
**Pull request by msulprizio**
### Name and Institution (Required)

Name: Melissa Sulprizio
Institution: Harvard

### Describe the update

Bingqing Zhang (Georgia Tech) has developed global continental chlorine emissions. The complete emission files from 1960-2014 in the netCDF format and a readme file have been published online at https://doi.org/10.7910/DVN/23BRII.

### Expected changes

This update will add a source of HCl to the full-chemistry simulations.

### Reference(s)

-   Zhang, B., & Liu, P., et al., Global Emissions of Hydrogen Chloride and Particulate Chloride from Continental Sources, Environmental Science & Technology, 56(7), 3894-3904, https://doi.org/10.1021/acs.est.1c05634, 2022.

### CODE REVIEW

File: run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem [hunk 1]
```diff
@@ -1372,6 +1372,22 @@ ORDONEZ_CH2ICL kg/m2/s Y Y F2000-%m2-01T00:00:00 none none CH2ICl ./HcoDir/IODIN
...
+HCL_WST  kg/m2/s Y Y F%y4-%m2-01T00:00:00 none none HCl_wstop ./HcoDir/GT_Chlorine/v2024-05/GT_Chlorine_01_01_$YYYY_V1.0.0.nc
+HCL_BIO  kg/m2/s Y Y F%y4-%m2-01T00:00:00 none none HCl_bbop  ./HcoDir/GT_Chlorine/v2024-05/GT_Chlorine_01_01_$YYYY_V1.0.0.nc
+HCL_AGR  kg/m2/s Y Y F%y4-%m2-01T00:00:00 none none HCl_agri  ./HcoDir/GT_Chlorine/v2024-05/GT_Chlorine_01_01_$YYYY_V1.0.0.nc
+PCL_ENE  kg/m2/s Y Y F%y4-%m2-01T00:00:00 none none pCl_ene   ./HcoDir/GT_Chlorine/v2024-05/GT_Chlorine_01_01_$YYYY_V1.0.0.nc
+PCL_IND  kg/m2/s Y Y F%y4-%m2-01T00:00:00 none none pCl_ind   ./HcoDir/GT_Chlorine/v2024-05/GT_Chlorine_01_01_$YYYY_V1.0.0.nc
+PCL_RES  kg/m2/s Y Y F%y4-%m2-01T00:00:00 none none pCl_res   ./HcoDir/GT_Chlorine/v2024-05/GT_Chlorine_01_01_$YYYY_V1.0.0.nc
+PCL_WST  kg/m2/s Y Y F%y4-%m2-01T00:00:00 none none pCl_wstop ./HcoDir/GT_Chlorine/v2024-05/GT_Chlorine_01_01_$YYYY_V1.0.0.nc
+PCL_BIO  kg/m2/s Y Y F%y4-%m2-01T00:00:00 none none pCl_bbop  ./HcoDir/GT_Chlorine/v2024-05/GT_Chlorine_01_01_$YYYY_V1.0.0.nc
+PCL_AGR  kg/m2/s Y Y F%y4-%m2-01T00:00:00 none none pCl_agri  ./HcoDir/GT_Chlorine/v2024-05/GT_Chlorine_01_01_$YYYY_V1.0.0.nc
```
Comment (yantosca) [inline run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem:1388]: For ExtData.rc, the `$YYYY` has to be `%y4` instead.

**Review by yantosca — CHANGES_REQUESTED** [review]
Thanks @msulprizio, this looks great.  I only have a couple of issues:
1. Fix formatting in ExtData.rc as noted in comment
2. We also need to add the `EmisHCl_Total` and `EmisHCl_Anthro` into the GCHP `Emissions` collection in `HISTORY.rc`.

File: run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem (hunk 1 above)
Comment (msulprizio) [reply run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem]: Thanks for catching this. It's not fixed in https://github.com/geoschem/geos-chem/pull/2275/commits/f13458d00134b900ee4c4d52b69424c76b51bc2a.

**Comment by msulprizio** [comment]
All GCClassic and GCHP integrations now pass

**Pull request merged**